### ISSUE TITLE
Support for Unity WebGL .data packages.

### ISSUE THREAD

**Issue by Irastris** [RaduMC/UnityStudio#15]
Hello. I'm hoping it'd be possible to add support for .data packages from Unity WebGL games.

I recently ripped a game from Newgrounds using this platform. All-in-all I ended up with 4 files:
- webgl.data
- webgl.js
- webgl.mem
- UnityLoader.js

Here is a random game using WebGL you could use for testing (I'm not posting the one I actually downloaded due to adult content being present): http://www.newgrounds.com/portal/view/671971

Direct link to the WebGL game: http://uploads.ungrounded.net/alternate//964000//964503_alternate_30851_r1.zip/

You can use Inspect Element to grab the 4 files from the page.

So, any possibility of adding support?

**Comment by Mactastic1-5** [comment]
Yes, it is possible to add support for Unity WebGL, but not in this project. Try a different program: [Google Search](http://lmgtfy.com/?q=unity+extract)
